clang/test/Sema/attr-availability-shadermodel.hlsl

```diff
@@ -0,0 +1,140 @@
+// RUN: %clang_cc1 -triple dxil-pc-shadermodel5.0-compute -fsyntax-only -verify %s
```

Comment:
could we do different runs for pixel and mesh to cover their different version availabilities from compute?

Reply:
Done.